uppsrc/ide/Builders/Install.cpp

```diff
@@ -172,7 +172,7 @@ void CreateBuildMethods()
 		SaveFile(bm_path, bm);
 	}
 #else
-	bool openbsd = ToLower(Sys("uname")).Find("openbsd") >= 0;
+	bool openbsd = ToLower(HostSys("uname")).Find("openbsd") >= 0;
```

Comment:
Bug: HostSys is not available in this place. Host::CMDLINE_PREFIX should be used. I will rework this.